Microsoft/BotFramework-Emulator
# Added the ability to open a bot from just a URL

This PR Fixes #1209 by adding the ability to connect to a bot by just a URL


# To Test
1. pull down this branch and build as normal then run the Emulator from sources
2. On the welcome page or the left hand panel or the file menu, choose open bot.
** Observe ** A new UI dialog is presented with the ability to type in a URL, browse for a bot file or choose from a list of previously opened bots.
3. Without opening a bot file, type a url in the input and choose *open/connect*
** Observe ** A new chat tab is is opened with a connection to the specified url.
